ssl/statem/statem_srvr.c

```diff
@@ -3653,6 +3735,28 @@ int tls_construct_server_certificate(SSL_CONNECTION *s, WPACKET *pkt)
     return 1;
 }
 
+#ifndef OPENSSL_NO_COMP_ALG
+int tls_construct_server_compressed_certificate(SSL_CONNECTION *sc, WPACKET *pkt)
+{
+    int alg = sc->ext.compress_certificate_from_peer;
+    OSSL_COMP_CERT *cc = sc->s3.tmp.cert->comp_cert[alg];
```

Comment:
An ossl_assert that this is non null might be a good idea.

Reply:
This code changed a bit...

Reply:
This comment is unresolved

Reply:
I thought the code had changed enough to no longer require it, but I added it in.

Reply:
`get_compressed_certificate_alg()` already uses `sc->s3.tmp.cert->comp_cert[alg]` as part of it's check, so it shouldn't be necessary.